plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -44,6 +42,22 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
 		},
 	)
+	SchedulingAlgorithmPredicateEvaluationDuration = prometheus.NewHistogram(
```

Comment:
@gmarek are you suggesting split the metric by specific predicate? Mind to explain?